Lesson Modules/Module 6- Data Binning.ipynb

```diff
@@ -101,7 +101,7 @@
     "    binnedCPM = np.array(sumCPM) / np.array(DataCount)          # np.array allows us to perform element-by-elemtent division\n",
     "    \n",
     "    avgError = np.array(sumError) / np.array(DataCount)\n",
-    "    stdError = avgError / np.sqrt(len(DataCount))               # standard error is average error divided by sqrt(# of elements)\n",
+    "    stdError = avgError / np.sqrt(DataCount)               # standard error is average error divided by sqrt(# of elements)\n",
```

Comment:
I think with the way .ipynv works, I'll have to rerun the script and submit another pull request to change the plot.